packages/evo-marko/src/tags/evo-avatar/component.ts

```diff
@@ -0,0 +1,44 @@
+import type { WithNormalizedProps } from "../../global";
```

Comment:
This whole component file should be removed, right? I think all of the logic was moved into `index.marko`